### PR TITLE
[Communication] - CHANGELOG - updating to include changes from previous pushes 

### PR DESCRIPTION
**Pull request by JoshuaLai**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,17 @@
 # Release History
 
+## 1.0.0-beta7 (unreleased)
+
+### New Features
+- Azure Communication
```
Comment (JoshuaLai) [inline CHANGELOG.md:6]: Should this be Azure Communication or be more specific: `Azure Communication Authentication`?

Comment (bsiegel) [reply CHANGELOG.md:6]: I think "Azure Communication Common Library" would make sense.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,17 @@
...
+  - Added new identifier  `MicrosoftTeamsUserIdentifier`. 
+
+### Breaking Changes
+- Azure Communication 
+  - Replaced `CommunicationUserCredential` with `CommunicationTokenCredential`.
+  - Previously `CommunicationTokenCredential` protocol renamed to `CommunicationTokenCredentialProviding`.
+  - Appended `Identifier` to all classes that conform to `CommunicationIdentifier`
```
Comment (bsiegel) [inline CHANGELOG.md:13]: I would favor a more conversational tone over brevity here. For example:

> All types that conform to `CommunicationIdentifier` now use the suffix `Identifier`. For example, the `PhoneNumber` class used to represent a phone number identifier is now named `PhoneNumberIdentifier`.

File: CHANGELOG.md [hunk 3]
```diff
@@ -24,7 +36,7 @@
   - Parameter labels are now mandatory for all API calls from Swift applications.
   - Except for the first parameter, parameter labels are now mandatory for all other parameters to delegate methods in
     Swift applications.
-  - And exception is now thrown if an application tries to render video/camera twice.
+  - An exception is now thrown if an application tries to render video/camera twice.
```
Comment (bsiegel) [inline CHANGELOG.md:39]: Unrelated, but I added this fix because it was bugging me :p

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,17 @@
 # Release History
 
+## 1.0.0-beta7 (unreleased)
+
+### New Features
+- Azure Communication Common Library
+  - Added new identifier  `MicrosoftTeamsUserIdentifier`.
```
Comment (bsiegel) [inline CHANGELOG.md:7]: Maybe explain what it's used for?

File: CHANGELOG.md [hunk 5]
```diff
@@ -1,5 +1,17 @@
 # Release History
 
+## 1.0.0-beta7 (unreleased)
```
Comment (bsiegel) [inline CHANGELOG.md:3]:
```suggestion
## 1.0.0-beta.7 (Unreleased)
```

**Review by bsiegel — APPROVED** [review]
:+1::sparkles:Ship It:sparkles::shipit:

Please remember to clean up the squashed commit message during the squash and merge process.

**Pull request merged**